spec/support/fetchable_behavior.rb

```diff
@@ -1,26 +1,27 @@
 shared_examples_for 'a record with a fetchable url' do
+  describe 'schema' do
+    it { is_expected.to have_db_column(:last_crawl_status).of_type(:string) }
+    it { is_expected.to have_db_column(:last_crawled_at).of_type(:datetime) }
+  end
+
   describe 'validations' do
-    it { is_expected.to validate_presence_of :url }
+    it { is_expected.to validate_presence_of(:url) }
     it { is_expected.to allow_value("http://some.site.gov/url").for(:url) }
-    it { is_expected.to allow_value("http://some.site.mil/").for(:url) }
-    it { is_expected.to allow_value("http://some.govsite.com/url").for(:url) }
-    it { is_expected.to allow_value("http://some.govsite.us/url").for(:url) }
-    it { is_expected.to allow_value("http://some.govsite.info/url").for(:url) }
-    it { is_expected.to allow_value("https://some.govsite.info/url").for(:url) }
-
     it 'limits the url length to 2000 characters' do
       record = described_class.new(valid_attributes.merge(url: ('x' * 2001) ))
       expect(record).not_to be_valid
       expect(record.errors[:url]).to include("is too long (maximum is 2000 characters)")
     end
 
-    context 'when the url extension is blacklisted' do
-      let(:movie_url) { "http://www.nps.gov/some.mov" }
-      let(:record) { described_class.new(valid_attributes.merge(url: movie_url)) }
+    context 'when last_crawl_status is > 255 characters' do
+      let(:temp) { described_class.create!(valid_attributes.merge(last_crawl_status: 'x' * 300)) }
```

Comment:
@ethanewing , I just noticed this. Any reason you named this variable `temp`?

Also, remember to avoid using `create` unless absolutely necessary, as persisting objects to the database slows down tests. I'll clean this up to simply validate but not save the record.

Reply:
I think it was just a bad name I gave it. I have an old habit of naming variables that are only used once or within a small scope "temp" and I forgot to change it to something more descriptive. I'll be better about that and the unnecessary `create`s

Reply:
Ah, that makes sense, but certainly something to avoid in the future. (In the Ruby world, it's SOP to use almost obnoxiously descriptive method & variable names.)